velox/common/memory/StreamArena.cpp

```diff
@@ -24,6 +24,19 @@ void StreamArena::newRange(int32_t bytes, ByteRange* range) {
   VELOX_CHECK_GT(bytes, 0);
   const memory::MachinePageCount numPages =
       memory::AllocationTraits::numPages(bytes);
+  // If new range 'bytes' is larger than the largest class page size, then we
```

Comment:
Maybe update comments for newRange() method in .h

Reply:
Updated.